docs/reference/tab-widgets/inference-api/infer-api-task.asciidoc

```diff
@@ -178,6 +178,30 @@ Also, when using this model the recommended similarity measure to use in the
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/google_vertex_ai_embeddings <1>
+{
+    "service": "googlevertexai",
+    "service_settings": {
+        "service_account_json": "<service_account_json>", <2>
+        "model_id": "text-embedding-004", <3>
+        "location": "<location>", <4>
+        "project_id": "<project_id>" <5>
+    }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The task type is `text_embedding` in the path and the `inference_id` which is the unique identifier of the {infer} endpoint is `google_vertex_ai_embeddings`.
```

Comment:
```suggestion
<1> The task type is `text_embedding` per the path. `google_vertex_ai_embeddings` is the unique identifier of the {infer} endpoint (its `inference_id`).
```
- just suggestion to split long sentence
- could also make the example something like `google_vertex_unique_id` so it's abundantly obvious